### DISCUSSION TITLE
When to support Android studio

### DISCUSSION THREAD

**Discussion by LiangLuDev** [community/community#7175] [Copilot Conversations] [answered]
as in the title

**Comment by jansorg** [comment, marked as answer]
The current canary build (Chipmunk) is supported, i.e. build branch 2021.2 and later.

**Reply by LiangLuDev** [reply]
When will the stable version support it?

**Reply by jansorg** [reply]
What I meant is it will be supported for the stable version as soon as Chipmunk has been promoted stable. I'm not sure when this will happen, though. Current Android Studio stable is based on an older version of IntelliJ and thus not yet supported.

**Reply by LiangLuDev** [reply]
thank you

**Reply by Subtixx** [reply]
So my issue with this is flutter does not support canary version / chipmunk. And copilot does not support stable. So I have to pick to either use flutter or copilot..
